README.md

```diff
@@ -97,16 +104,21 @@ create virtual machines for hosting the containers.
 
 ##### Kernel
 
-The [`linux`](https://github.com/kata-containers/linux) kernel is used by the hypervisor to boot the guest image.
+The [Linux\*](https://github.com/kata-containers/linux) kernel is used by the hypervisor to boot the guest image.
 
 ### Documentation
 
-The [`documentation`](https://github.com/kata-containers/documentation)
+The [documentation](https://github.com/kata-containers/documentation)
 repository hosts documentation common to all code components.
 
+### Packaging
+
+The [packaging](https://github.com/kata-containers/packaging)
+repository is used for creating packages for system components.
```

Comment:
optionally add something like 'including rootfs images and linux kernels' maybe.

Reply:
Done.